sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -200,8 +202,9 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
           logInfo("Skip tests because old Spark versions don't support Java 21.")
         }
       } else {
-        logError(s"Python version <  ${TestUtils.minimumPythonSupportedVersion}, " +
-          "the running environment is unavailable.")
+        logError(
```

Comment:
We don't need to migrate the test changes.

Reply:
Oh, I see.